src/Backend/Modules/Pages/Ajax/UploadFile.php

```diff
@@ -23,11 +23,17 @@ public function execute(): void
     {
         $request = $this->getRequest();
 
-        $fileName = $this->writeFile(
-            $this->getFileContentFromRequest($request),
-            $this->getFileNameFromRequest($request),
-            $request->get('type')
-        );
+        try {
+            $fileName = $this->writeFile(
+                $this->getFileContentFromRequest($request),
+                $this->getFileNameFromRequest($request),
+                $request->get('type')
```

Comment:
These check warnings by codecov are quite annoying while code reviewing files without unit tests. I wonder if we can disable that somehow

Reply:
the patch ones are disabled so I don't know why they are showing

Reply:
4bf38b485a718091613a7889ce78d3b650e19de9